interfaces/cython/cantera/test/utilities.py

```diff
@@ -70,7 +70,7 @@ def tearDownClass(cls):
 
     def assertNear(self, a, b, rtol=1e-8, atol=1e-12, msg=None):
         cmp = 2 * abs(a - b)/(abs(a) + abs(b) + 2 * atol / rtol)
-        if cmp > rtol:
+        if not cmp < rtol:
```

Comment:
I can never remember the priority order of `not` versus comparisons. I think it'd make sense to group the comparison for clarity, although it's not required. Also if you can add a comment here as to why the comparison is done this way, I think that'd help too. The previous version is more intuitive.

Reply:
Oops - should have waited longer before merging. I thought the logic made sense. I can add the comment in #1101.